volatility3/framework/automagic/symbol_cache.py

```diff
@@ -492,6 +492,21 @@ def get_identifiers(self, operating_system: Optional[str]) -> List[bytes]:
         return output
 
 
+def load_cache_manager(cache_file: Optional[str] = None) -> CacheManagerInterface:
+    """Loads a cache manager based on a specific cache file"""
+    if cache_file is None:
+        cache_file = os.path.join(constants.CACHE_PATH, constants.IDENTIFIERS_FILENAME)
+    # Different implementations of cache
+    if not os.path.exists(cache_file):
+        raise ValueError("Non-existant cache file provided")
+    with open(cache_file, "rb") as fp:
+        header = fp.read(4)
+        if header not in [b"SQLi"]:
```

Comment:
It's reading 4 bytes and comparing against 4 bytes, not sure why the `header not in [b"SQLi"]`? shouldn't it be `header != b"SQLi"`? or one-liner `open(cache_file, "rb").read(4) != b"SQLi"`?

Reply:
This is all for future extensibility, when we get different file formats for the cache.  The 4 byte header check will likely be used for whatever format we end up with, so I coded it with essentially a case statement with only one clause in it...